server/rpc.go

```diff
@@ -310,6 +312,14 @@ func GetTranscoder(context context.Context, orch Orchestrator, req *TranscoderRe
 	return &tr, nil
 }
 
+type serverSegmentResponse struct {
```

Comment:
Yes, you're right, will use proto where it will be needed, but probably in this place I'll throw out structured reply any way and will let it be like it is now